bazel/repository_locations.bzl

```diff
@@ -1222,10 +1222,9 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Chrome URL parsing library",
         project_desc = "Chrome URL parsing library",
         project_url = "https://quiche.googlesource.com/googleurl",
-        # Static snapshot of https://quiche.googlesource.com/googleurl/+archive/dd4080fec0b443296c0ed0036e1e776df8813aa7.tar.gz
         version = "dd4080fec0b443296c0ed0036e1e776df8813aa7",
         sha256 = "59f14d4fb373083b9dc8d389f16bbb817b5f936d1d436aa67e16eb6936028a51",
-        urls = ["https://storage.googleapis.com/quiche-envoy-integration/{version}.tar.gz"],
+        urls = ["https://quiche.googlesource.com/googleurl/+archive/{version}.tar.gz"],
```

Comment:
I could be wrong, but I think this doesn't work because the sha256 hash changes on each fetch of the file because of some timestamp in the file which is not static. But if it works, LGTM!

Reply:
yeah I think that's exactly what happened :-(
tempted to host  my own download on my chromium account for a temporary fix while we get the bucket fixed - maybe if we haven't sorted by tomorrow